package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "generator-assets",
-  "version": "2.5.0-dev",
+  "version": "2.6.0",
   "description": "Asset generation plug-in for Photoshop Generator",
   "main": "main.js",
   "generator-core-version": "^3.4.0",
```

Comment:
should probably change this to ^3.6 since it relies on that version to work as advertised.